spec/main-process/appwindow-spec.js

```diff
@@ -0,0 +1,240 @@
+'use strict';
+
+var assert = require('assert');
+var sinon = require('sinon');
+var SandboxedModule = require('sandboxed-module');
+
+describe('AppWindow', function () {
+  var sandbox;
+  var AppWindow;
+  var fakeDialog;
+  var fakeBrowserWindow;
+  var fakeShell;
+  var appWindow;
+  var fakeKillChildProcess;
```

Comment:
Do not use scope variables. See http://yeoman.io/contributing/testing-guidelines.html (Never use scoped variables)

I know, I did the same mistake in the existing tests :blush: